spec/features/review_card_spec.rb

```diff
@@ -4,12 +4,17 @@
 
 DatabaseCleaner.strategy = :truncation
 
+
 describe "the review card process" do
   before do
     DatabaseCleaner.clean
+    Capybara.register_driver :rack_test do |app|
+      Capybara::RackTest::Driver.new(app, :headers => { 'HTTP_USER_AGENT' => 'Capybara' })
```

Comment:
Line is too long. [90/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.